cpp/rte/mo_optical_props.h

```diff
@@ -159,33 +159,55 @@ class OpticalProps {
 
   // Are the bands of two objects the same? (same number, same wavelength limits)
   bool bands_are_equal(OpticalProps const &rhs) const {
-    if ( this->get_nband() != rhs.get_nband() || this->get_nband() == 0) { return false; }
-    yakl::ScalarLiveOut<bool> ret(true);
-    // for (int j=1 ; j <= size(this->band_lims_wvn,2); j++) {
-    //   for (int i=1 ; i <= size(this->band_lims_wvn,1); i++) {
-    auto &this_band_lims_wvn = this->band_lims_wvn;
-    auto &rhs_band_lims_wvn  = rhs.band_lims_wvn;
-    parallel_for( Bounds<2>( size(this->band_lims_wvn,2) , size(this->band_lims_wvn,1) ) , YAKL_LAMBDA (int j, int i) {
-      if ( abs( this_band_lims_wvn(i,j) - rhs_band_lims_wvn(i,j) ) > 5*epsilon(this_band_lims_wvn) ) {
-        ret = false;
+    // if ( this->get_nband() != rhs.get_nband() || this->get_nband() == 0) { return false; }
```

Comment:
Can you drop a comment here explaining why this section is commented out in favor of the host version below?